R/html_dependency.R

```diff
@@ -366,11 +369,11 @@ copyDependencyToDir <- function(dependency, outputDir, mustWork = TRUE) {
   }
 
   srcfiles <- file.path(dir, files)
-  if (any(!file.exists(srcfiles))) {
+  if (any(bad <- !file.exists(srcfiles))) {
     stop(
       sprintf(
         "Can't copy dependency files that don't exist: '%s'",
```

Comment:
I think the assignment in an `if()` can be pretty confusing, so this should be moved out.

GitHub wouldn't let me add this suggestion in one block (something about not allowing suggestions with deleted lines) so it's in two parts.

```suggestion
  missing_srcfiles <- !file.exists(srcfiles)
  if (any(missing_srcfiles)) {
    stop(
      sprintf(
        "Can't copy dependency files that don't exist: '%s'",
```